src/component/Tooltip.tsx

```diff
@@ -104,10 +104,7 @@ const tooltipDefaultProps: TooltipProps<number, string> = {
   allowEscapeViewBox: { x: false, y: false },
   reverseDirection: { x: false, y: false },
   offset: 10,
-  // TODO: replace invalid properties
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  viewBox: { x1: 0, x2: 0, y1: 0, y2: 0 },
+  viewBox: { x: 0, y: 0, height: 0, width: 0 },
```

Comment:
zeros should be good defaults. If Tooltip doesn't go through `generateCategoricalChart` it won't get rendered and doesn't need a viewBox anyways